From c0faa221bfb713b2287ed2f101285d56f28ba091 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 12:56:02 +0800
Subject: [PATCH] do not support network setting for now

We have a long term plan to support it, but not now.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Refactor for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/core/configuration/system.py | 3 ++-
 pyanaconda/installation.py              | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/core/configuration/system.py b/pyanaconda/core/configuration/system.py
index 365fbd539..19be89fbb 100644
--- a/pyanaconda/core/configuration/system.py
+++ b/pyanaconda/core/configuration/system.py
@@ -140,7 +140,8 @@ class SystemSection(Section):
     @property
     def can_configure_network(self):
         """Can we configure the network?"""
-        return self._is_boot_iso or self._is_booted_os
+        # Do not support network config for now
+        return False
 
     @property
     def can_detect_unsupported_hardware(self):
diff --git a/pyanaconda/installation.py b/pyanaconda/installation.py
index 2458bca63..0548b539b 100644
--- a/pyanaconda/installation.py
+++ b/pyanaconda/installation.py
@@ -126,7 +126,7 @@ def _prepare_configuration(payload, ksdata):
     configuration_queue.append(os_config)
 
     # schedule network configuration (if required)
-    if conf.target.can_configure_network and conf.system.provides_network_config:
+    if False:
         overwrite = payload.type in PAYLOAD_LIVE_TYPES
         network_config = TaskQueue("Network configuration", N_("Writing network configuration"))
         network_config.append(Task("Network configuration",
-- 
2.7.4

